src/test/groovy/org/prebid/server/functional/tests/TargetingSpec.groovy

```diff
@@ -453,26 +455,26 @@ class TargetingSpec extends BaseSpec {
         def response = defaultPbsService.sendAmpRequest(ampRequest)
 
         then: "Response should contain targeting hb_pb"
-        assert response.targeting["hb_pb"] == getRoundedTargetingValueWithDefaultPrecision(1.50)
+        assert response.targeting["hb_pb"] == String.format("%,.2f", max.setScale(2, RoundingMode.DOWN))
     }
 
-    def "PBS auction should make right calculation for hb_pb targeting"() {
+    def "PBS auction should use ranges.max value for hb_pb targeting when bid.price biggest that ranges.max"() {
```

Comment:
Maybe `PBS auction should use ranges.max value for hb_pb targeting when bid.price is greater than ranges.max`?